static/app/components/projectPageFilter.tsx

```diff
@@ -94,9 +92,12 @@ export function ProjectPageFilter({router, specificProjectSlugs, ...otherProps}:
     ? projects.filter(project => specificProjectSlugs.includes(project.slug))
     : projects;
 
-  const [_, otherProjects] = partition(specifiedProjects, project => project.isMember);
+  const [memberProjects, otherProjects] = partition(
+    specifiedProjects,
+    project => project.isMember
+  );
 
-  const isSuperuser = isActiveSuperuser();
+  const {isSuperuser} = ConfigStore.get('user');
```

Comment:
not sure if this is now the wrong way to check for a superuser, but this was what was used in the pageFilters/container.tsx file and so to maintain consistency I've copied it over here.